### PR TITLE
fix: Auto-approve rust-usage-accountant

### PR DESCRIPTION
**Pull request by untitaker**

### CODE REVIEW

File: .github/workflows/auto-approve.yml [hunk 1]
```diff
@@ -13,6 +13,7 @@ jobs:
         startsWith(github.event.issue.title, 'publish: getsentry/pytest-sentry@') ||
         startsWith(github.event.issue.title, 'publish: getsentry/relay/py@') ||
         startsWith(github.event.issue.title, 'publish: getsentry/responses@') ||
+        startsWith(github.event.issue.title, 'publish: getsentry/rust-usage-accountant@') ||
```
Comment (fpacifici) [inline .github/workflows/auto-approve.yml:16]: Could you please add the python version as well ? 
https://github.com/getsentry/usage-accountant/

Comment (untitaker) [reply .github/workflows/auto-approve.yml:16]: done

**Review by asottile-sentry — APPROVED** [review]

**Comment by untitaker** [comment]
@asottile-sentry please merge as well, i have no access to this repo at all (even the PR is from my fork)

**Pull request merged**